README.md

```diff
@@ -30,23 +37,30 @@ Snapshotter talks to CSI over socket (/run/csi/socket by default, configurable b
   * If the `VolumeSnapshotRef` is set to nil, skip this content since it is not bound to any snapshot object.
   * Otherwise, the controller verifies whether the content object is correctly bound to a snapshot object. In case the `VolumeSnapshotRef.UID` is set but it does not match its snapshot object or snapshot no long exists, the content object and its associated snapshot will be deleted.
 
-## Usage
+## Deployment
```

Comment:
Do we also want to provide some recommended RBACs for users (not sidecar, not controller) to operate on the CRs?

Reply:
Does that include admins?  Users can only see VolumeSnapshot, Admins can see both.

Reply:
Correct, I mean provide different recommended rbacs for different personas